modules/fedimint-ln-client/src/lib.rs

```diff
@@ -232,6 +232,7 @@ pub struct LightningOperationMetaPay {
     pub change: Vec<OutPoint>,
     pub is_internal_payment: bool,
     pub contract_id: ContractId,
+    pub gateway_id: Option<secp256k1::PublicKey>,
```

Comment:
TIL: [you don't need `#[serde(default)]` annotation for `Option<T>` fields :exploding_head: ](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=9739232131409ded8e1d6e6f2199ea88)